app/Integrations/Immich/ImmichPlugin.php

```
@@ -0,0 +1,475 @@
+            'access_token' => null,
+            'refresh_token' => null,
+            'expiry' => null,
+            'refresh_expiry' => null,
+            'auth_metadata' => [
+                'server_url' => null,
+                'api_key' => null,
+            ],
+        ]);
+    }
+
+    public function createInstance(IntegrationGroup $group, string $instanceType, array $initialConfig = [], bool $withMigration = false): Integration
```

PHPMD (2.15.0)

Avoid unused parameters such as '$withMigration'. (undefined)

(UnusedFormalParameter)